### DISCUSSION TITLE
Interaction with ChickenSys Translator (like SCSI2SD)

### DISCUSSION THREAD

**Discussion by janczmok1970** [ZuluSCSI/ZuluSCSI-firmware#349] [Q&A]
How can i read out the Device Info for ChickenSys Translater

They have a scsi2sd interface, it asks for the scsi2sd-util who read outs direct information from scsi2sd

How can i read out the partition info thru USB on zuluscsi-rp ?

Cheers
Jan

**Comment by chickeneps** [comment]
Did you create an image of your ENTIRE SCSI2SD'fied SD card? Is that the imge you are referring to that you want to use with ZuluSCSI?

**Comment by aperezbios** [comment]
No response from requestor ever, closing.
